scripts/compile-mlibc-sysdeps.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+SYSDEPS=${3:-"dripos lemon aero ironclad vinix lyre"}
+cd pkg-builds/
+for sysdep in $(echo $SYSDEPS); do
+	if [ ! -d "mlibc-sysdep-$sysdep" ]; then
+		meson setup \
+			"-Dc_args=['-fno-stack-protector', '-U__linux__']" \
+			"-Dcpp_args=['-fno-stack-protector', '-U__linux__']" \
+			-Dbuild_tests=true \
+			-Db_sanitize=undefined \
+			"-Dlinux_kernel_headers=$2" \
+			--cross-file $1/ci/$sysdep.cross-file mlibc-sysdep-$sysdep $1
```

Comment:
quoting is missing, also prefer ${} for cases like $foo.bar, since that's a lot more readable in such contexts